core/src/Streamly/Data/Stream/Zip.hs

```diff
@@ -0,0 +1,33 @@
+-- |
+-- Module      : Streamly.Data.Stream.Zip
+-- Copyright   : (c) 2017 Composewell Technologies
+--
+-- License     : BSD3
+-- Maintainer  : streamly@composewell.com
+-- Stability   : experimental
+-- Portability : GHC
+--
+module Streamly.Data.Stream.Zip
+    (
+      ZipStream (..)
+    , ZipSerialM
+    , ZipSerial
+    , zipWithM
+    , zipWith
```

Comment:
Remove these two, these are already exported from Data.Stream. This module is only for the newtype wrapper for Applicative instance.